flink-ml-lib/src/test/java/org/apache/flink/ml/feature/PolynomialExpansionTest.java

```diff
@@ -149,44 +135,56 @@ public void testOutputSchema() {
 
     @Test
     public void testSaveLoadAndTransform() throws Exception {
-        Normalizer normalizer =
-                new Normalizer().setInputCol("denseVec").setOutputCol("outputVec").setP(1.5);
+        PolynomialExpansion polynomialExpansion =
+                new PolynomialExpansion()
+                        .setInputCol("denseVec")
+                        .setOutputCol("outputVec")
+                        .setDegree(2);
 
-        Normalizer loadedNormalizer =
+        PolynomialExpansion loadedPolynomialExpansion =
                 TestUtils.saveAndReload(
-                        tEnv, normalizer, TEMPORARY_FOLDER.newFolder().getAbsolutePath());
+                        tEnv, polynomialExpansion, TEMPORARY_FOLDER.newFolder().getAbsolutePath());
 
-        Table output = loadedNormalizer.transform(inputDataTable)[0];
-        verifyOutputResult(output, loadedNormalizer.getOutputCol(), EXPECTED_DENSE_OUTPUT);
+        Table output = loadedPolynomialExpansion.transform(inputDataTable)[0];
+        verifyOutputResult(output, loadedPolynomialExpansion.getOutputCol(), EXPECTED_DENSE_OUTPUT);
     }
 
     @Test
     public void testInvalidP() {
         try {
-            Normalizer normalizer =
-                    new Normalizer().setInputCol("denseVec").setOutputCol("outputVec").setP(0.5);
-            normalizer.transform(inputDataTable);
+            PolynomialExpansion polynomialExpansion =
+                    new PolynomialExpansion()
+                            .setInputCol("denseVec")
+                            .setOutputCol("outputVec")
+                            .setDegree(-1);
+            polynomialExpansion.transform(inputDataTable);
             fail();
         } catch (Exception e) {
-            assertEquals("Parameter p is given an invalid value 0.5", e.getMessage());
+            assertEquals("Parameter degree is given an invalid value -1", e.getMessage());
         }
     }
 
     @Test
     public void testDenseTransform() throws Exception {
-        Normalizer normalizer =
-                new Normalizer().setInputCol("denseVec").setOutputCol("outputVec").setP(1.5);
-
-        Table output = normalizer.transform(inputDataTable)[0];
-        verifyOutputResult(output, normalizer.getOutputCol(), EXPECTED_DENSE_OUTPUT);
+        PolynomialExpansion polynomialExpansion =
```

Comment:
Could you please add test cases when degree > 3 and when vector size > 3? These test cases can help improve the coverage rate.